### PR TITLE
[DOCS] How to connect to a MySQL database

### PR DESCRIPTION
**Pull request by Aylr**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
-
-
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [x] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+from ruamel import yaml
+
+import great_expectations as ge
+from great_expectations.core.batch import BatchRequest, RuntimeBatchRequest
+
+CONNECTION_STRING = "mysql+pymysql://root@localhost/test_ci"
+# CONNECTION_STRING = "mysql+pymysql://root@127.0.0.1:8889/test_ci"  # TODO REMOVE THIS and UPDATE LINE NUMBERS
+
```
Comment (Shinnnyshinshin) [inline tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py]: I think you intended to change this..? :)

File: docs/guides/connecting_to_your_data/database/mysql.md [hunk 2]
```diff
@@ -1,4 +1,161 @@
...
+</TabItem>
+<TabItem value="python">
+
+Put your connection string in this template:
+
+```python file=../../../../tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py#L19-L36
+```
+Run this code to test your configuration.
+```python file=../../../../tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py#L42
+```
```
Comment (Shinnnyshinshin) [inline docs/guides/connecting_to_your_data/database/mysql.md:87]: This needs to be updated I think

File: docs/guides/connecting_to_your_data/database/mysql.md [hunk 3]
```diff
@@ -1,4 +1,161 @@
...
+## Additional Notes
+
+To view the full scripts used in this page, see them on GitHub:
+
+- [mysql_yaml_example.py](https://github.com/great-expectations/great_expectations/blob/develop/tests/integration/docusaurus/connecting_to_your_data/database/mysql_yaml_example.py)
+- [mysql_python_example.py](https://github.com/great-expectations/great_expectations/blob/develop/tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py)
+
+## Next Steps
+
+<NextSteps />
```
Comment (Shinnnyshinshin) [inline docs/guides/connecting_to_your_data/database/mysql.md:161]: The first link `How to create and edit Expectations with instant feedback from a sample Batch of data` was not rendering correctly on the `deploy-preview-2070` build for me... are you seeing something similar on your end?

**Review by Shinnnyshinshin — CHANGES_REQUESTED** [review]
Thank you very very much for this. Looks great other than 2 small changes

File: tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py (hunk 1 above)
Comment (Aylr) [reply tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py]: aaaack yes thank you.

File: docs/guides/connecting_to_your_data/database/mysql.md (hunk 3 above)
Comment (Aylr) [reply docs/guides/connecting_to_your_data/database/mysql.md:161]: Funny - I noticed that on a different branch I was on earlier. I'll check it out.

Comment (Aylr) [reply docs/guides/connecting_to_your_data/database/mysql.md:161]: fixed

File: docs/guides/connecting_to_your_data/database/mysql.md (hunk 2 above)
Comment (Aylr) [reply docs/guides/connecting_to_your_data/database/mysql.md:87]: Fixed

File: tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py (hunk 1 above)
Comment (Aylr) [reply tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py]: Fixed

**Review by Shinnnyshinshin — APPROVED** [review]
Looks wonderful :) thank you!

**Pull request merged**